src/main/java/de/medizininformatikinitiative/torch/util/Slicing.java

```diff
@@ -22,6 +26,30 @@ public class Slicing {
 
     private static final Logger logger = LoggerFactory.getLogger(Slicing.class);
 
+    /**
+     * Checks if the given element is a sliced element and returns the sliced element definitions mapped
+     * to the applicable structureDefinition otherwise empty map.
+     *
+     * @param base     HAPI Base (Element) which should be checked
+     * @param contexts ElementId with its associated Profiles
+     * @return Returns null if no slicing is found and an ElementDefinition for the slice otherwise
```

Comment:
on top, you write it will return an empty map and here you write it returns null